lib/jwt/jwt.go

```diff
@@ -200,6 +200,29 @@ func (k *Key) SignSnowflake(p SignParams, issuer string) (string, error) {
 	return k.sign(claims)
 }
 
+type PROXYSignParams struct {
+	ClusterName        string
+	SourceAddress      string
+	DestinationAddress string
+}
+
+const expirationPROXY = time.Second * 60
+
+// SignPROXY will create short lived signed JWT that is used in signed PROXY header
+func (k *Key) SignPROXY(p PROXYSignParams) (string, error) {
+	claims := Claims{
+		Claims: jwt.Claims{
+			Subject:   fmt.Sprintf("%s/%s", p.SourceAddress, p.DestinationAddress),
```

Comment:
One wonders if it would make sense semantically for the subject to be `p.SourceAddress` and the audience to be `p.DestinationAddress`  ?